packer_templates/sles/http/sles-15-sp1-x86_64-autoinst.xml

```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0"?>
```

Comment:
Is there a reason we couldn't just have 15 and 15 SP1 use the same file?

Reply:
There is no reason other than to conform to the file structure of the repo as is. So surely it is possible to use this AutoYaST for both 15.0 and 15.1